core/backend/src/CloudSqlite.ts

```diff
@@ -265,18 +266,21 @@ export namespace CloudSqlite {
     /**
      * Disconnect this CloudContainer from its CloudCache. There must be no open databases from this container. Leaves the container attached to the
      * CloudCache so it is available for future sessions.
+     * @note This function does nothing if the CloudContainer is not connected to a CloudCache. @see connect
      */
     disconnect(): void;
 
     /**
      * Permanently Detach and Disconnect this CloudContainer from its CloudCache. There must be no open databases from this container.
+     * @note This function does nothing if the CloudContainer is not connected to a CloudCache. @see connect
      */
     detach(): void;
 
     /**
      * Poll cloud storage for changes from other processes. *No changes* made by other processes are visible to
      * this CloudContainer unless/until this method is called.
      * @note this is automatically called whenever the write lock is obtained to ensure all changes are against the latest version.
+     * @note This function does nothing if the CloudContainer is not connected to a CloudCache. @see connect
```

Comment:
This function actually throws an error if the CloudContainer is not connected to a cloudcache. You can see "pollManifest" in c++ code has the function RequireConnected() in it. Maybe instead of @note, it should be @throws in this case. 

Some of the others may also call 'requireConnected'